testsuite/forge/src/backend/k8s/helm-values/aptos-node-default-values.yaml

```diff
@@ -32,5 +32,12 @@ service:
     enableRestApi: true
     enableMetricsPort: true
 
+# to give HAProxy access to more resources, schedule it on validators nodegroup
+# in the long term, we should consider giving the "utilities" nodegroup more resources
+# or removing it from the reference implementation entirely
+haproxy:
```

Comment:
This will be pretty expensive for us - essentially running haproxy on 16 vcore machine. Why is this needed? `haproxy` was hardly using any resources last time I checked.

Reply:
This was an experiment. These changes are not strictly needed.